test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,24 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe('Fizz')
```

Comment:
A few more test cases would be good here, that way you know you havent just picked a single correct answer and it works in multiple instances